demisto_sdk/commands/common/update_id_set.py

```diff
@@ -588,6 +588,7 @@ def get_layout_data(path):
     pack = get_pack_name(path)
     incident_indicator_types_dependency = {id_}
     incident_indicator_fields_dependency = get_values_for_keys_recursively(json_data, ['fieldId'])
+    definition_id = json_data.get('definitionId')
```

Comment:
very cool idea but please let's handle a case where definition id might hold `incident` or `indicator` as this will create incorrect connections everywhere - true for all changes in id_set

Also, we currently do not keep the definitionId in the struct for the mappers, classifiers and so on - we need to add them to the schema.

Reply:
Handled inside the find_dependencies function. https://github.com/demisto/demisto-sdk/blob/4a7f2e67de3326f37cc3d5591bc23b5c716aefe1/demisto_sdk/commands/find_dependencies/find_dependencies.py#L575 .

Reply:
still need to change the schemas as discussed to make sure this field is not deleted